source/images/homepage/transparency.svg

```diff
@@ -0,0 +1,53 @@
+<svg class="LogoTransparency" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 302 302" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
CSS doesn't, but the JS animation code uses some of these classes to find elements

This particular one is not used, but it's children (`.LogoTransparency-piece`) still are. And I guess removing the parent while keeping the children wouldn't be right